test/jest/helpers/utils.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable import/prefer-default-export */
+export const componentPropsCheck = (Component, testId, expectedProps, partialCompare = false) => {
+  const propertiesForCompare = Component.mock.calls
+    .reverse()
+    .find(item => item[0]['data-testid'] === testId);
+
+  if (partialCompare) {
+    expect(propertiesForCompare[0]).toStrictEqual(expect.objectContaining(expectedProps));
```

Comment:
Maybe it is better
const resultExpectedProps = partialCompare
    ? objectContaining(expectedProps)
    : {
      ...expectedProps,
      'data-testid': testId,
    };

Reply:
Done. Thanks!